modules/security-deploy-secrets.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * security/container_security/security-deploy.adoc
+
+[id="security-deploy-secrets_{context}"]
+= Secrets and ConfigMaps
+
+The `Secret` object type provides a mechanism to hold sensitive information such
+as passwords, {product-title} client configuration files, `dockercfg` files,
+and private source repository credentials. Secrets decouple sensitive content
+from pods. You can mount secrets into containers using a volume plug-in or the
+system can use secrets to perform actions on behalf of a pod.
+
+For example, to add a secret to your deployment configuration using the web
+console so that it can access a private image repository:
+
+. Create a new project.
+
+. Navigate to `Resources -> Secrets` and create a new secret. Set `Secret Type` to
```

Comment:
Any GUI item should be in asterisks for bold.

```
Navigate to *Resources* -> *Secrets* and...
```
(Note the arrows are not inside bold)

Reply:
Okay. Done.